File: Web/RMA-Member-Portal/memberportal-app/src/app/shared/common/payee-type.ts
import { BaseClass } from "./base-class";

export class PayeeType extends BaseClass {
    payeeTypeId: number;
    name: string;
    description: string;
    isSundry: boolean;
    isMedical: boolean;
    isPD: boolean;
    isFatal: boolean;
    isDaysOff: boolean;
    isFuneralBenefit: boolean;
    isPension: boolean;
  }

File: Service/Admin/RMA.Service.Admin.MasterDataManager.Contracts/Entities/PayeeType.cs
using RMA.Common.Entities;

namespace RMA.Service.Admin.MasterDataManager.Contracts.Entities
{
    public class PayeeType : AuditDetails
    {
        public int PayeeTypeId { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public bool IsSundry { get; set; }
        public bool IsMedical { get; set; }
        public bool IsPd { get; set; }
        public bool IsFatal { get; set; }
        public bool IsDaysOff { get; set; }
        public bool IsFuneralBenefit { get; set; }
        public bool IsPension { get; set; }
    }
}
